tests/python/unittest/test_loss.py

```diff
@@ -225,7 +225,7 @@ def test_ctc_loss_train():
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 10
 
 
-@with_seed(1234)
+@with_seed()
```

Comment:
nit: please refer to https://github.com/apache/incubator-mxnet/pull/11888/files#diff-cb652780258e73a9cd08568f38929aa2R2509 for adding some tracking for this test.

Reply:
Added the tracking for this